### DISCUSSION TITLE
Trying to give arguments via cmd

### DISCUSSION THREAD

**Discussion by Nightchanger** [ppy/osu#21487] [Q&A] [answered]
I'm using playnite to manager my games, I'm trying to add an install action that downloads the file and gives a directory argument.

C:\...\osu!install.exe -d "\my directory"

Can this be done?

**Comment by bdach** [comment, marked as answer]
Judging by the `osu!install.exe` filename you're asking about stable. This repository is about lazer only.

**Reply by peppy** [reply]
Also of note, we won't be adding any functionality like this to stable, so you're out of luck on this for now.
